libmisc/salt.c

```diff
@@ -12,33 +12,98 @@
 #ident "$Id$"
 
 #include <sys/time.h>
+#include <crypt.h>
+#include <string.h>
 #include <stdlib.h>
 #include <stdio.h>
 #include <assert.h>
 #include "prototypes.h"
 #include "defines.h"
 #include "getdef.h"
 
+#if (defined CRYPT_GENSALT_IMPLEMENTS_AUTO_ENTROPY && \
+     CRYPT_GENSALT_IMPLEMENTS_AUTO_ENTROPY)
+/* Maximum size of the setting for crypt_gensalt(). */
```

Comment:
This comment "Maximum size of the setting for crypt_gensalt()." looks irrelevant here.

Reply:
Ahh, yes, that somehow slipped into there…  Will remove with next push.

Reply:
Fixed.